integrated_channels/canvas/client.py

```diff
@@ -183,28 +236,33 @@ def delete_course_completion(self, user_id, payload):  # pylint: disable=unused-
         pass
 
     # Private Methods
-
-    def _update_course_details(self, course_id, serialized_data):
+    def _update_course_details(self, course_id, course_details):
         """
         Update a course for image_url (and possibly other settings in future)
-        This is used only for settings that are not settable in the initial course creation
+        Also sets course to 'offer' state by sending 'course[event]=offer'
+
+        Arguments:
+          - course_details (dict): { 'image_url' } : optional used if present to course[image_url]
```

Comment:
if you're going to add one of the params to the docstring I'd at least add the other even if it's straightforward as to not confuse anyone?